R-proj/src/sample_points.cpp

```diff
@@ -20,7 +20,7 @@
 #include "sampling/sampling.hpp"
 
 template <typename Polytope, typename RNGType, typename PointList, typename NT, typename Point>
-void sample_from_polytope(Polytope &P, RNGType &rng, PointList &randPoints, unsigned int const& walkL, unsigned int const& numpoints,
+void sample_from_polytope(Polytope &P, int type, RNGType &rng, PointList &randPoints, unsigned int const& walkL, unsigned int const& numpoints,
```

Comment:
This is implemented in this branch https://github.com/AlexManochis/volume_approximation/tree/ellipsoid_uniform .
However I didn't merge it into this branch because it is not clear which PR is going to be merged first.

Reply:
it seems that the current is more mature, and probably will be merged first. If you want you can merge that branch  `https://github.com/AlexManochis/volume_approximation/tree/ellipsoid_uniform` here.

Reply:
I didnt merge it but I created an enum to declare a random walk